static/js/client.js

```diff
@@ -1338,10 +1360,12 @@ var app = {}, browserSettings = {}, browserStorage = $.localStorage;
       currentAlarmType = alarm.type;
       console.info('generating timeAgoAlarm', alarm.type);
       $('#container').addClass('alarming-timeago');
-      if (level === 'warn') {
-        generateAlarm(alarmSound);
+      console.log('ago:', ago);
+      var message = {'title': 'Last data received ', 'message': ago.value + ago.label};
+      if (level == 'warn') {
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Expected '===' and instead saw '=='.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/769587981/issues/source?bid=2452379&fileBranchId=2457543#l1365)